spec/built-in-modules/color.md

```diff
@@ -593,7 +594,24 @@ This function is also available as a global function named `mix()`.
 saturate($color, $amount)
 ```
 
-* Throw an error.
+* If this function is called as a global function:
+
+  * If this function is called with a single argument `$amount`:
+
+    * If `$amount` is a number or a [special variable string], return a plain
+      CSS function string with the name `"saturate"` and the argument `$amount`.
+
+  * Otherwise:
+
+    * If `$color` isn't a color, throw an error.
+
+    * If `$amount` isn't a number, throw an error.
+
+    * Return the result of calling `changeHsl($saturation)`, where `$saturation`
+      is the `$color`'s saturation incremented by `$amount` clamped between the
+      inclusive range `[0, 100]`.
```

Comment:
Leaving this out was intentional—the old behavior is deprecated, and the canonical spec only includes non-deprecated behavior.